application/core/LSYii_Validators.php

```diff
@@ -70,7 +70,10 @@ protected function validateAttribute($object, $attribute)
         if ($this->xssfilter) {
             $object->$attribute = $this->xssFilter($object->$attribute);
             if ($this->isUrl) {
-                $object->$attribute = str_replace('javascript:', '', $object->$attribute);
+                // If the URL contains "javascript:", we clear the field completely. The URL is either valid o invalid.
+                if (stripos($object->$attribute, 'javascript:') !== false) {
```

Comment:
without expression : `JaVaScript:alert(1)` `jav&#x0D;ascript:alert('XSS');` or `&#X6A;avascript:alert(1)` 

and a lot more